ddtrace/contrib/redis/util.py

```diff
@@ -19,6 +19,7 @@ def _extract_conn_tags(conn_kwargs):
             net.TARGET_HOST: conn_kwargs["host"],
             net.TARGET_PORT: conn_kwargs["port"],
             redisx.DB: conn_kwargs.get("db") or 0,
+            redisx.CLIENT_NAME: conn_kwargs.get("client_name") or "",
```

Comment:
thinking about this a little more, I wonder if we should only include this if it is set?


e.g.

```python
conn_tags = {
    net.TARGET_HOST: conn_kwargs["host"],
    net.TARGET_PORT: conn_kwargs["port"],
    redisx.DB: conn_kwargs.get("db") or 0,
}

client_name = conn_kwargs.get("client_name")
if client_name:
    conn_tags[redisx.CLIENT_NAME] = client_name

return conn_tags
```

What do you think about this?

Reply:
it probably comes down to:

1. How often are people likely setting this?
2. The increase in bytes ingested by sending the empty tag

Reply:
Since this is an optional setting that makes sense. Also, this attribute is not supported on old redis-py versions.

Reply:
ah, good point!